### DISCUSSION TITLE
Welcome to embedded-cassandra Discussions!

### DISCUSSION THREAD

**Discussion by nosan** [nosan/embedded-cassandra#167] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nkostiv** [comment]
Hi, found your great tool today. Don`t know where is the better place to put such information.

During startup received next JVM error
```
A fatal error has been detected by the Java Runtime Environment:
#
#  EXCEPTION_ACCESS_VIOLATION (0xc0000005) at pc=0x0000000010014ed4, pid=25664, tid=0x00000000000048e8
# Java VM: Java HotSpot(TM) 64-Bit Server VM (25.271-b09 mixed mode windows-amd64 compressed oops)
# Problematic frame:
# C  [sigar-amd64-winnt.dll+0x14ed4]
```

The reason was the JDK version and sigar-amd64-winnt.dll do not match. Only one option was to lower the JDK version to jdk1.8.0_241.

**Reply by nosan** [reply]
Hi, unfortunately I can't fix this because it is an issue in Apache Cassandra itself. I think it would be better to raise an issue in Apache Cassandra jira.

**Reply by nkostiv** [reply]
Yes, I know. Just wanted to mention this as solution - fix tip.

**Reply by nosan** [reply]
Thanks a lot!
